src/model/formToDats.js

```diff
@@ -63,7 +65,10 @@ class FormToDats {
         return Object.assign(pp, {
           dates: pp.dates.map((date) => {
             return Object.assign(date, {
-              date: format(date.date, 'yyyy-MM-dd') + ' 00:00:00'
+              date: format(date.date, 'yyyy-MM-dd') + ' 00:00:00',
+              type: {
+                value: 'publication date'
```

Comment:
This sets the type of every date for a primary publication to 'publication date'. Is that intended?

Reply:
in theory, there should always be a type associated to the date. I did not realize that there could be an array of dates for the publication as well. So I guess the best would be to add a type box next to the date field for the publications as well. Right?

Reply:
The type box is there already (the label says description)